### PR TITLE
[SHELL32] Improve callbacking of CDefView

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose

JIRA issue: N/A

- Delete `DefMessageSFVCB` method.
- Add some callbacking.

### CODE REVIEW

File: dll/win32/shell32/CDefView.cpp [hunk 1]
```diff
@@ -115,7 +115,7 @@ class CDefView :
     private:
         HRESULT _MergeToolbar();
         BOOL _Sort();
-        HRESULT _DoFolderViewCB(UINT uMsg, WPARAM wParam, LPARAM lParam);
+        HRESULT CallCB(UINT uMsg, WPARAM wParam, LPARAM lParam);
```
Comment (learn-more) [inline dll/win32/shell32/CDefView.cpp:118]: Why the rename?

File: dll/win32/shell32/CDefView.cpp [hunk 2]
```diff
@@ -2466,6 +2472,11 @@ HRESULT WINAPI CDefView::DestroyViewWindow()
 
     if (m_hWnd)
     {
+        HWND hwndTemp = m_hWnd;
+        m_hWnd = NULL;
+        CallCB(SFVM_WINDOWCLOSING, (WPARAM)hwndTemp, 0);
+        m_hWnd = hwndTemp;
```
Comment (learn-more) [inline dll/win32/shell32/CDefView.cpp:2478]: Why un-assign m_hWnd here?

File: dll/win32/shell32/CDefView.cpp [hunk 3]
```diff
@@ -3541,15 +3552,7 @@ HRESULT CDefView::_MergeToolbar()
     return S_OK;
 }
 
-// The default processing of IShellFolderView callbacks
-HRESULT CDefView::DefMessageSFVCB(UINT uMsg, WPARAM wParam, LPARAM lParam)
-{
-    // TODO: SFVM_GET_CUSTOMVIEWINFO, SFVM_WINDOWCREATED
-    TRACE("CDefView::DefMessageSFVCB uMsg=%u\n", uMsg);
-    return E_NOTIMPL;
-}
```
Comment (learn-more) [inline dll/win32/shell32/CDefView.cpp:3550]: Why was this removed?

File: dll/win32/shell32/CDefView.cpp (hunk 1 above)
Comment (katahiromz) [reply dll/win32/shell32/CDefView.cpp]: Reverted.

File: dll/win32/shell32/CDefView.cpp (hunk 2 above)
Comment (katahiromz) [reply dll/win32/shell32/CDefView.cpp]: OK. Simplified.

File: dll/win32/shell32/CDefView.cpp (hunk 3 above)
Comment (katahiromz) [reply dll/win32/shell32/CDefView.cpp:3550]: This is introduced by my uncertain idea. There is no default processing.
